system/views/view.py

```diff
@@ -0,0 +1,84 @@
+import tkinter as tk
+from tkinter import ttk
+
+
+class View(tk.Tk):
+
+    def __init__(self, *args, **kwargs):
+        tk.Tk.__init__(self,  *args, **kwargs)
+
+        # creating a container
+        container = tk.Frame(self)
+        container.pack(side="top", fill="both", expand=True)
+
+        container.grid_rowconfigure(0, weight=1)
+        container.grid_columnconfigure(0, weight=1)
+
+        # initializing frames to an empty array
+        self.frames = {}
+
+        # iterating through a tuple consisting
+        # of the different page layouts
+        for F in (RegistrationPage, LoginPage):
+            frame = F(container, self)
+
+            # initializing frame of that object from
+            # startpage, page1, page2 respectively with
+            # for loop
+            self.frames[F] = frame
+
+            frame.grid(row=0, column=0, sticky="nsew")
+
+        self.show_frame(RegistrationPage)
+
+        # to display the current frame passed as
+        # parameter
+
+    def show_frame(self, cont):
+        frame = self.frames[cont]
+        frame.tkraise()
+
+
+class RegistrationPage(tk.Frame):
```

Comment:
Not entirely sure if these classes should be broken down into multiple .py files. Leave it for now.